zerver/lib/markdown/api_arguments_table_generator.py

```diff
@@ -91,11 +91,9 @@ def render_table(self, arguments: Sequence[Mapping[str, Any]]) -> List[str]:
         # TODO: Fix naming now that this no longer renders a table.
         table = []
         argument_template = """
-<div>
-    <a href="#{argument}"></a>
-</div>
+
 <div class="api-argument" id="{argument}">
-    <p class="api-argument-name"><strong>{argument}</strong> {required} {deprecated}</p>
+    <p class="api-argument-name"><strong>{argument}</strong> {required} {deprecated}<a href="#{argument}" class="api-argument-hover-link"><i class="fa fa-chain" style="color:#000"></i></a></p>
```

Comment:
Looks great! Can you move ` style="color:#000"` to portico.scss too?

Reply:
Sure, I will do that.